### DISCUSSION TITLE
TurkishDroid

### DISCUSSION THREAD

**Discussion by aaytaccc** [community/community#32878] [Mobile] [answered]
_Hi GitHub Mobile Team.
Thanks for putting all your effort to make this app. I think it could support users more with additional feature for Actions/Workflows status view. Look forward to hear your opinion about that._

**Aytaç ÇAKIR******

**Comment by stevepopovich** [comment, marked as answer]
Thanks for the feedback! I have passed this on to the team. We will likely have something for this in the future. Thanks!
